### ISSUE TITLE
UX/UI: adding a reference image for each Page

### ISSUE THREAD

**Issue by carloposo** [Requarks/wiki#1242]
In the documentation of wikijs (almost) each page title is accompanied by an image, like the following.

<img width="1001" alt="Screenshot 2019-11-20 at 16 43 10" src="https://user-images.githubusercontent.com/1691849/69253901-ac8b0700-0bb5-11ea-8413-0a51705a21a8.png">

Is there any chance we can get the same feature?

**Comment by XerqJon** [comment]
You _can_ do that.

Anywhere on the page (I always put it on the bottom) you can use the following syntax:
`![icon-topright](https://imgurl.com){.align-abstopright}`

**Comment by carloposo** [comment]
Awesome.
Can I do it in the WYSIWYG editor, too?

**Comment by XerqJon** [comment]
If i had to guess I would say not with the visual editor, at least not until a specific action is added to assign the appropriate class.

You could accomplish it by editing the HTML file manually and adding:
`<img src="https://imgURL.com" class="align-abstopright">`

**Comment by carloposo** [comment]
Correct me if I am wrong: that's only possible if I initially chose "Code" as the editor for my Page...

**Comment by XerqJon** [comment]
@carloposo what I meant was either edit the .html file in your local storage or, if you're also syncing to git, edit there. 
With the WYSIWYG editor, you'll be limited to only the features made available by toolbar button at the top or if the option to edit source within the editor is made available. 

I would recommend using the markdown editor. As far as I can tell, it's the most versatile because you can also insert normal HTML into it and it will render properly.
If you aren't familiar with markdown let me just say I wasn't familiar with markdown before using v1 of the wiki but it's easy to catch on and there is a lot of reference material. The markdown editor by default has a preview pane on the right which shows you what the page is going to look like.

**Comment by carloposo** [comment]
@XerqJon thanks, mate.
I'll look into markdown. At the moment, I am sticking to the Visual Editor because in the future editors of my wiki will use that, being non-technical people. So I am testing that for them.

Would be nice to have a HTML editor button in the toolbar of the Visual Editor. I'll open a request.

Carlo

**Comment by XerqJon** [comment]
Sure thing.
You can add a feature request here: https://wiki.js.org/feedback/

**Comment by carloposo** [comment]
@XerqJon Just done. Thanks.
